src/sentry/static/sentry/app/actionCreators/organization.jsx

```diff
@@ -36,6 +36,9 @@ export async function fetchOrganizationDetails(api, slug, detailed, silent) {
     setActiveOrganization(org);
 
     if (detailed) {
+      // TODO(davidenwang): Change these to actions after organization.projects
```

Comment:
Will we ever be able to change the org details API completely? I suspect there are external consumers that would break. We could stop using the heavy endpoints internally though.

Reply:
Ahh yes we will be forced to maintain that endpoint for external users, but internally the goal is to only ever fetch the lighweight version of org details.